src/ripple/app/paths/impl/PaySteps.cpp

```diff
@@ -423,6 +429,12 @@ toStrandV2 (
         if (hasCurrency && hasIssuer &&
             isXRP(pe.getCurrency()) != isXRP(pe.getIssuerID()))
             return {temBAD_PATH, Strand{}};
+
+        if (hasIssuer && (pe.getIssuerID() == noAccount()))
```

Comment:
Should the Issue currency/account consistency be checked as above?
Not sure if this is correct ...
```       if (hasIssuer && !isConsistent({ pe.getCurrency(), pe.getIssuerID()}))
            return {temBAD_PATH, Strand{}};```